packages/api/src/graphql/invoice.ts

```diff
@@ -0,0 +1,110 @@
+import {Invoice, InvoiceItem, InvoiceSort} from '../db/invoice'
+import {Context} from '../context'
+import {
+  GraphQLID,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLList,
+  GraphQLInt,
+  GraphQLInputObjectType,
+  GraphQLEnumType
+} from 'graphql'
+import {GraphQLDateTime} from 'graphql-iso-date'
+import {GraphQLUser} from './user'
+import {createProxyingResolver} from '../utility'
+import {GraphQLRichText} from './richText'
+import {GraphQLPageInfo} from './common'
+
+export const GraphQLInvoiceItem = new GraphQLObjectType<InvoiceItem, Context>({
+  name: 'InvoiceItem',
+  fields: {
+    createdAt: {type: GraphQLNonNull(GraphQLDateTime)},
+    modifiedAt: {type: GraphQLNonNull(GraphQLDateTime)},
+    name: {type: GraphQLNonNull(GraphQLString)},
+    description: {type: GraphQLRichText},
+    quantity: {type: GraphQLNonNull(GraphQLInt)},
+    amount: {type: GraphQLNonNull(GraphQLInt)},
```

Comment:
amount here is payment amount? Did you intend it to be int?

Reply:
@penk85 i don't understand the question. Why should it be something different than an integer?